app/src/main/res/values/component_colors.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="oppia_primary_color">@color/primary_color</color>
```

Comment:
Some of these just repeat variants in color_palette. That's not the intention of component_colors.

Think of it this way:
- color_defs defines the actual hex colors (only used by color_palette)
- color_palette defines a list of generic colors that are used everywhere through component colors (only used by component_colors)
- component_colors defines more specific colors (either colors only used by a single component, similar to strings & dimensions, or generic colors that don't need to be switched out by night mode)

Reply:
Removed from component_colors